floor.go

```diff
@@ -92,32 +95,62 @@ func (f *Floor) watchFloorPrice() {
 	f.close = make(chan int, 1)
 
 	// watch floor price
+	var oldPrice float64
+	var increase bool
+	var priceString string
 	for {
 
 		select {
 		case <-f.close:
 			logger.Infof("Shutting down price watching for %s/%s", f.Marketplace, f.Name)
 			return
 		case <-ticker.C:
-			price, activity, err := utils.GetFloorPrice(f.Marketplace, f.Name)
+			price, activity, currency, err := utils.GetFloorPrice(f.Marketplace, f.Name)
 			if err != nil {
 				logger.Errorf("Error getting floor rates: %s\n", err)
 				continue
 			}
 
+			// Use platform currency if not set.
+			if f.Currency == "" {
+				f.Currency = currency
+			}
+
+			// Convert price to string format.
+			if f.Currency == "ETH" {
+				priceString = fmt.Sprintf("Ξ %s", strconv.FormatFloat(price, 'f', -1, 64))
+			} else {
+				priceString = fmt.Sprintf("%s %s", strconv.FormatFloat(price, 'f', -1, 64), f.Currency)
```

Comment:
I moved the code of [e089130](https://github.com/rssnyder/discord-stock-ticker/commit/e089130cc1ff216d958b9802bb018323c3c2f6f3) out of the `GetFloorPrice` function.  Please check if it still does what it is supposed to do.